computer-vision/README.md

```diff
@@ -0,0 +1,42 @@
+2. For implementation of some of the concepts, [OpenCV](https://opencv.org/releases/) will help you. Note that you are required to have some background in python. There are many tutorials available, such as [the one by Corey Schafer](https://www.youtube.com/watch?v=YYXdXT2l-Gg&list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU), or [by sentdex](https://www.youtube.com/playlist?list=PLQVvvaa0QuDe8XSftW-RAxdo6OmaeL85M). 
+
+## Basic Roadmap
+
+1. To learn basic concepts that are used in image processing and computer vision, [Introduction to Computer Vision](https://www.udacity.com/course/introduction-to-computer-vision--ud810) is a good place to start. The course starts with introducing various image processing techniques required, before moving on to introducing camera geometry, such as the pinhole model, intrinsic and extrinsic camera parameters and analysing the point correspondences between two views of the same scene. The course moves on to introduce the concept of features, scale invariance and then introducing the state of the art method, Scale Invariant Feature Transform (SIFT). One can complete the whole course in order to get an idea on different computer vision and image processing techniques. 
+2. For discussing the feature detection and matching techniques, one can have a look at the [documentation](https://docs.opencv.org/3.4.2/db/d27/tutorial_py_table_of_contents_feature2d.html) provided by OpenCV. The documentation starts with the introduction to the concept of features, followed by discussing various techniques to obtain the feature points. 
+
+## Other Lecture Series
+
+To look for more advanced concepts, one can refer to the following lecture courses selectively.
+
+1. [Computer Vision for Visual Effects, by Rich Radke, Rensselaer Polytechnique Institute](https://www.youtube.com/playlist?list=PLuh62Q4Sv7BUJlKlt84HFqSWfW36MDd5a)
+2. [Photogrammetry Course by Cyrill Stachniss, University of Bonn](https://www.youtube.com/playlist?list=PLgnQpQtFTOGRsi5vzy9PiQpNWHjq-bKN1)
+3. [Multiple View Geometry by Daniel Cremers, Technical University Munich](https://www.youtube.com/playlist?list=PLTBdjV_4f-EJn6udZ34tht9EVIW7lbeo4)
+4. [Computer Vision by Mubarak Shah, University of Central Florida](https://www.youtube.com/playlist?list=PLd3hlSJsX_Imk_BPmB_H3AQjFKZS9XgZm)
+
+## Other Books
+
+1. [Computer Vision: Algorithms and Applications, Richard Szeliski](http://szeliski.org/Book/drafts/SzeliskiBook_20100903_draft.pdf)
+2. [Multiple View Geometry in computer vision, Richard Hartley and Andrew Zisserman](http://cvrs.whu.edu.cn/downloads/ebooks/Multiple%20View%20Geometry%20in%20Computer%20Vision%20(Second%20Edition).pdf)
+3. [Digital image processing, Rafael C. Gonzalez](http://web.ipac.caltech.edu/staff/fmasci/home/astro_refs/Digital_Image_Processing_2ndEd.pdf)
```

Comment:
A fundamental book for IP.
And a fundamental video lecture series for Image and video processing: https://www.youtube.com/watch?v=bxhJEe38bhY&list=PLZ9qNFMHZ-A79y1StvUUqgyL-O0fZh2rs